website/docs/components/form/masked-input/partials/guidelines/guidelines.md

```diff
@@ -68,6 +68,12 @@ For shorter, simpler forms, indicate optional fields instead.
   <F.Label>Label</F.Label>
 </Hds::Form::MaskedInput::Field>
 
+## Copyable masked input 
+
+The `MaskedInput` component lets you show a Copy Button by setting the `hasCopyButton` property to `true` Users can interact with this button to copy the `MaskedInput's` content to their clipboard. Refer to the [copy button](www.helios.com/copy/copy-button) usage guidelines to learn more about copying content. 
```

Comment:
Couple of broad notes here:
- Generally when referring to the component, we've moved away from wrapping the name in backticks and instead reserve that styling for properties, arguments, etc.
- When linking to another document/page you don't need the entire URL, all you need is the local path, which in this case is `/components/copy/copy-button`.

```suggestion
The Masked Input component lets you show a Copy Button by setting the `hasCopyButton` property to `true` Users can interact with this button to copy the Masked Input's content to their clipboard. Refer to the [Copy Button](/components/copy/copy-button) usage guidelines to learn more about copying content. 
```

Reply:
Additionally, I might recommend re-wording this a little bit. Maybe consider:

Setting the `hasCopyButton` property to `true` will display a [Copy Button](/components/copy/copy-button) adjacent to the Masked Input, allowing users to copy the value in the Masked Input to their clipboard. Refer to the [Copy Button](/components/copy/copy-button) usage guidelines for more details around copying content.

Reply:
Ah, okay! Thank you for clarifying.